src/components/EventModal.vue

```diff
@@ -6,15 +6,15 @@
 				<input v-model="eventTitle" type="text">
 			</div>
 			<div class="dateTimePicker">
-				<DatetimePicker v-model="startDate"
+				<DateTimePickerNative v-model="startDate"
 					:format="dateFormat"
 					:clearable="false"
 					:minute-step="5"
 					:show-second="false"
```

Comment:
this attributes should be removed / replaced. Native datetime picker does't have some of them / or have other analogs